### PR TITLE
add conn's address info in Response

### PR DESCRIPTION
**Pull request by xuecai**
for #535

### CODE REVIEW

File: http.go [hunk 1]
```diff
@@ -281,6 +287,31 @@ func (w *requestBodyWriter) Write(p []byte) (int, error) {
 	return len(p), nil
 }
 
+func (resp *Response) parseNetConn(conn net.Conn) {
+	if conn.RemoteAddr() != nil {
+		if resp.raddr == nil {
+			resp.raddr = make([]byte, 0)
```
Comment (erikdubbelboer) [inline http.go]: You can safe one allocation by already reserving the amount of bytes you're going to need below.

**Comment by erikdubbelboer** [comment]
If you rebase on master, I have just [added some tests to make sure we have 0 allocations](https://github.com/valyala/fasthttp/commit/61039c93c2a31d3be661c50269d5800090f44175).

**Comment by xuecai** [comment]
I learned a lot. Thanks for allocations test.
i'm sorry for reset commits, i made a mistake when I  rebase code

**Comment by xuecai** [comment]
I think I have done it in [Response.Reset()](https://github.com/valyala/fasthttp/pull/537/files#diff-31dcde2645a9daeff4e0064653331de4R846)?

**Comment by erikdubbelboer** [comment]
Sorry I missed that.

@kirillDanshin what do you think? If you agree then you can merge it.

**Comment by xuecai** [comment]
@erikdubbelboer @kirillDanshin i'm sorry, i find a bug on response copy and i fix it. 
please reivew.

**Comment by xuecai** [comment]
i find CopyTo miss tests in fasthttp.
may i add then by another pr later?

**Comment by xuecai** [comment]
hi, Is there any other advice for this pr? @erikdubbelboer @kirillDanshin

**Comment by erikdubbelboer** [comment]
Thanks.

**Pull request merged**